src/main/java/frc/robot/RobotContainer.java

```diff
@@ -55,132 +59,239 @@ public class RobotContainer extends LightningContainer {
+            new Trigger(() -> (coralCollector.getVelocity() < 0)).whileTrue(leds.enableState(LED_STATES.CORAL_COLLECT));
+        
+            // climber.setDefaultCommand(new RunCommand(() ->
+            // climber.setPower(MathUtil.applyDeadband(-copilot.getLeftY(),
+            // ControllerConstants.JOYSTICK_DEADBAND)), climber));
 
-        // more sim stuff
-        if (Robot.isSimulation()){
-                // climber.setDefaultCommand(new RunCommand(() -> climber.setPower(copilot.getLeftY())));
+            rod.setDefaultCommand(new SetRodState(rod, RodStates.STOW));
+            
+            new Trigger(() -> rod.onTarget()).whileFalse(leds.enableState(LED_STATES.ROD_MOVING));
         }
+
     }
 
     @Override
     protected void configureButtonBindings() {
+        // robot centric driving
+        new Trigger(() -> driver.getLeftTriggerAxis() > 0.25).whileTrue(drivetrain.applyRequest(DriveRequests
+            .getRobotCentric(
+                () -> MathUtil.applyDeadband(-driver.getLeftX(),
+                    ControllerConstants.JOYSTICK_DEADBAND),
+                () -> MathUtil.applyDeadband(-driver.getLeftY(),
+                    ControllerConstants.JOYSTICK_DEADBAND),
+                () -> MathUtil.applyDeadband(-driver.getRightX(),
+                    ControllerConstants.JOYSTICK_DEADBAND))));
         // sets slow mode
         new Trigger(() -> driver.getRightTriggerAxis() > 0.25)
-                .onTrue(new InstantCommand(() -> drivetrain.setSlowMode(true)))
-                .onFalse(new InstantCommand(() -> drivetrain.setSlowMode(false)));
-
-        // sets robot centric
-        new Trigger(() -> driver.getLeftTriggerAxis() > 0.25)
-                .whileTrue(drivetrain.applyRequest(
-                        DriveRequests.getRobotCentric(() -> -(driver.getLeftX() * drivetrain.getSpeedMult()),
-                        () -> -(driver.getLeftY() * drivetrain.getSpeedMult()),
-                        () -> -(driver.getRightX() * drivetrain.getTurnMult()))));
+            .onTrue(new InstantCommand(() -> drivetrain.setSlowMode(true)))
+            .onFalse(new InstantCommand(() -> drivetrain.setSlowMode(false)));
 
+        // drivetrain brake
         new Trigger(driver::getXButton).whileTrue(drivetrain.applyRequest(DriveRequests.getBrake()));
 
+        // reset forward
         new Trigger(() -> driver.getStartButton() && driver.getBackButton()).onTrue(
-                new InstantCommand(() -> drivetrain.seedFieldCentric()));
-                
-        // // TODO: Remove Standin Command
-        // new Trigger(() -> rod.onTarget()).whileTrue(leds.enableState(LED_STATES.ROD_ON_TARGET));
+            new InstantCommand(() -> drivetrain.seedFieldCentric()));
+
+        new Trigger(() -> copilot.getRightBumperButton())
+            .whileTrue(new RunCommand((() -> rod.setState(RodStates.SOURCE)), rod));
+        new Trigger(() -> copilot.getLeftBumperButton())
```

Comment:
use the `SetRodState` command and also format the trigger with a condensed lambda (`::`)